packages/dev/inspector-v2/src/components/properties/particles/attractorList.tsx

```diff
@@ -0,0 +1,109 @@
+import { GizmoManager } from "core/Gizmos";
+import { StandardMaterial } from "core/Materials";
+import { Color3 } from "core/Maths";
+import type { AbstractMesh } from "core/Meshes";
+import { Attractor } from "core/Particles";
+import type { ParticleSystem } from "core/Particles";
+import type { Scene } from "core/scene";
+import type { Nullable } from "core/types";
+import { useCallback, useEffect, useState, type FunctionComponent } from "react";
+import { Color3PropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/colorPropertyLine";
+import { SyncedSliderPropertyLine } from "shared-ui-components/fluent/hoc/propertyLines/syncedSliderPropertyLine";
+import { List } from "shared-ui-components/fluent/primitives/list";
+import type { ListItem } from "shared-ui-components/fluent/primitives/list";
+import { useResource } from "../../../hooks/resourceHooks";
+import { AttractorComponent } from "./attractor";
+type AttractorListProps = {
+    scene: Scene;
+    gizmoManager: GizmoManager;
+    attractors: Array<Attractor>;
+    system: ParticleSystem;
+};
+
+// For each Attractor, create a listItem consisting of the attractor and its debugging impostor mesh
+function AttractorsToListItems(attractors: Nullable<Array<Attractor>>) {
+    return (
+        attractors?.map((attractor, index) => {
+            return {
+                id: index,
+                data: attractor,
+                sortBy: 0,
+            };
+        }) ?? []
+    );
+}
+
+const CreateGizmoManager = (scene: Scene) => {
+    const gizmoManager = new GizmoManager(scene);
+    gizmoManager.positionGizmoEnabled = true;
+    gizmoManager.attachableMeshes = [];
+    return gizmoManager;
+};
+const CreateSharedMaterial = (scene: Scene, impostorColor: Color3) => {
+    const material = new StandardMaterial("Attractor impostor material", scene);
+    material.diffuseColor = impostorColor;
+    material.reservedDataStore = { hidden: true }; // Ensure scene explorer doesn't show the material
+    return material;
+};
+
+export const AttractorList: FunctionComponent<AttractorListProps> = (props) => {
+    const { scene, system } = props;
+    const [items, setItems] = useState<Array<ListItem<Attractor>>>([]);
+
+    // All impostors share a scale and material/color (for now!)
+    const [impostorScale, setImpostorScale] = useState(1);
+    const [impostorColor, setImpostorColor] = useState<Color3>(() => Color3.White());

```

Comment:
The Color3 type can be inferred from the value, just like in the previous line, so this could just be `useState(() => Color3.White());`